library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentViewHolder.java

```diff
@@ -149,7 +149,7 @@ private void configFileAttach() {
                 attachments.add(attachment);
             }
         }
-        AttachmentListAdapter attachAdapter = new AttachmentListAdapter(context, attachments, false, false);
+        AttachmentListAdapter attachAdapter = new AttachmentListAdapter(context, attachments, false, false, null);
```

Comment:
see previous comment about constructor overloading (if OK this code is not needed)